docs/reference/mapping/types/unsigned_long.asciidoc

```diff
@@ -0,0 +1,147 @@
+[role="xpack"]
+[testenv="basic"]
+
+[[unsigned-long]]
+=== Unsigned long data type
+++++
+<titleabbrev>Unsigned long</titleabbrev>
+++++
+
+Unsigned long is a numeric field type that represents an unsigned 64-bit
+integer with a minimum value of 0 and a maximum value of +2^64^-1+
+(from 0 to 18446744073709551615).
```

Comment:
In Elasticsearch range queries with the from/to notation, the "to" bound is exclusive, while I think it's inclusive in your example here. Maybe be explicit by saying "to XYZ included" ?